netlogo-gui/project/autogen/i18n/GUI_Strings_en.txt

```diff
@@ -165,6 +165,7 @@ tabs.info.help = Help
 
 tabs.code = Code
 tabs.code.rightclick.quickhelp = Quick Help
+tabs.code.rightclick.jumptodeclaration = Jump To Declaration
```

Comment:
I would prefer this a line higher or a line lower so it doesn't break up `tabs.code.rightclick.quickhelp` items.